modules/fluviewer.nf

```diff
@@ -130,12 +137,11 @@ process fluviewer {
     if [[ ! -f ${sample_id}_HA_consensus.fa ]]; then
         echo "HA segment consensus not generated. Skipping FindCleave.py..."
     else
-	python ${projectDir}/bin/FindCleave.py -i ${sample_id}_HA_consensus.fa -o ${sample_id}_HPAI.tsv
+	    python ${projectDir}/bin/FindCleave.py -i ${sample_id}_HA_consensus.fa -o ${sample_id}_HPAI.tsv
```

Comment:
I don't think this is a big problem but we shouldn't need to specify `${projectDir}/bin/FindCleave.py`. Nextflow should always put `${projectDir}/bin` on the `PATH` for each process, so (as long as the script is executable) you should just be able to say `FindCleave.py` and the system will know where to find it.

Reply:
I've updated this myself.